MoyaResponse.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  File.swift
```

Comment:
Opened an issue to remove them later: https://github.com/ashfurrow/Moya/issues/134